tests/mocks/database/population/data/nonClassMeetings.ts

```diff
@@ -0,0 +1,36 @@
+import { DAY } from '../../../../../src/common/constants';
```

Comment:
That's true that the top-level is the `NonClassParent`, but because there's the embedded meeting data there I thought it would be worth using a slightly different name (and I believe "non-class meetings" is what we call this combination in the front end). The meeting data is embedded here because each `nonClassEvent` or `courseInstance` should have a unique set of meetings, and splitting the meetings out makes it more likely that a meeting might be accidentally assigned to both a `nonClassEvent` and a `courseInstance`, which is not allowed. 

(tbh, this population data is all kinds of hacky and cobbled together, but the goal was to have something that's _mostly_ human readable in the files but easy enough to loop though and import into the database)

Reply:
Ah, fair enough - the nomenclature of this type of data in the app is/was messing with my head a bit - maybe we should revisit that in another ticket...